cmd/golangBuild.go

```diff
@@ -404,6 +429,58 @@ func reportGolangTestCoverage(config *golangBuildOptions, utils golangBuildUtils
 	return nil
 }
 
+func retrieveGolangciLint(utils golangBuildUtils, golangciLintURL, golangciLintDir string) error {
+	// installation instructions: https://golangci-lint.run/usage/install/#linux-and-windows
+	httpClient := &http.Client{
```

Comment:
please use the http package we provide:
https://github.com/SAP/jenkins-library/blob/master/pkg/http/downloader.go

This helps with resilience, testability, ...

Reply:
Added usage of ```SendRequest``` method of piper http package. We can use ```SendRequest``` without any change of ```golangBuildUtilsBundle``` and ```golangBuildUtils```. ```golangBuildUtilsBundle``` already implements the interface.

is it ok or should we use ```GetRequest``` method of piper http package?

Reply:
Added usage of ```DownloadFile```